ios/library/WhirlyGlobe-MaplyComponent/src/control/MaplyBaseInteractionLayer.mm

```diff
@@ -3664,7 +3659,58 @@ - (NSObject *)getSelectableObject:(WhirlyKit::SimpleIdentity)objId
 
 - (NSObject*)selectLabelsAndMarkerForScreenPoint:(CGPoint)screenPoint
 {
-    return nil;
+    return [[self selectMultipleLabelsAndMarkersForScreenPoint:screenPoint] firstObject];
+}
+
+- (NSMutableArray*)selectMultipleLabelsAndMarkersForScreenPoint:(CGPoint)screenPoint
+{
+    NSMutableArray *retSelectArr = [NSMutableArray array];
+    return retSelectArr;
+}
+
+- (NSMutableArray*)convertSelectedObjects:(std::vector<SelectionManager::SelectedObject>)selectedObjs
+{
+    NSMutableArray *retSelectArr = [NSMutableArray array];
+    if (!selectedObjs.empty())
```

Comment:
This isn't really necessary, as a for-loop over an empty collection is no slower.